src/Codeception/Module/Laminas.php

```diff
@@ -181,6 +181,16 @@ public function addServiceToContainer(string $name, object $service): void
         $this->client->addServiceToContainer($name, $service);
     }
 
+    /**
+     * Adds factory to a Laminas container
+     *
+     * @part services
+     */
+    public function addFactoryToContainer(string $name, $factory): void
```

Comment:
Correct. Docblock could be updated for $factory parameter. I didn't do that because other methods don't have params described either.
I can add test to https://github.com/Naktibalda/codeception-laminas-tests similar to https://github.com/Naktibalda/codeception-laminas-tests/blob/master/tests/functional/AddServiceToContainerCept.php.
General principle behind it is that whenever you use ServiceManager#build() to build a service, new service instance is returned. In order to have expected service stub built by the ServiceManager we can return it with help of substitution of factory for the service in tests.

Reply:
> I didn't do that because other methods don't have params described either.

That's because @TavoNiievez went through all methods and replaced `@param` annotations with scalar typehints.
`$factory` can't have typehint, so `@param` provides useful information for IDEs and static analysis tools.

> General principle behind it is that whenever you use ServiceManager#build() to build a service, new service instance is returned. In order to have expected service stub built by the ServiceManager we can return it with help of substitution of factory for the service in tests.

I'm not asking for myself, but for people who may want to use it. I tried searching yesterday, but failed to find anything relevant.